.changeset/rare-laws-deny.md

```diff
@@ -0,0 +1,10 @@
+---
+'app': major
+'backend': minor
+---
+
+Update backstage to [1.18.4](https://github.com/backstage/backstage/releases/tag/v1.18.4)
+
+**BREAKING**
+
+Updated Tekton plugin now expects a `janus-idp.io/tekton` annotation instead of a `janus-idp.io/tekton-enabled` annotation on the catalog entities.
```

Comment:
This should be part of another PR as not related to Backstage 1.18.x

Reply:
So do you want to revert the upgrades to the `@janus-idp` packages and only upgrade the `@backstage` packages?

Reply:
Reverted the commit https://github.com/Zaperex/backstage-showcase/commit/495bd3d3d1e061420e6d15ab24c20217f8fd311b